python/ray/includes/task.pxd

```diff
@@ -18,10 +18,13 @@ from ray.includes.unique_ids cimport (
 )
 
 
-cdef extern from "ray/raylet/task_execution_spec.h" namespace "ray::raylet" nogil:
-    cdef cppclass CTaskExecutionSpecification "ray::raylet::TaskExecutionSpecification":
+cdef extern from "ray/raylet/task_execution_spec.h" \
+        namespace "ray::raylet" nogil:
+    cdef cppclass CTaskExecutionSpecification \
```

Comment:
Can these `\`'s be removed?

Reply:
I don't think so, also we shouldn't obsess about removing \.